agbenchmark/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "workspace": "C:\\Users\\silen\\miniagi",
-  "func_path": "agent/benchmarks.py",
+  "workspace": "autogpt/workspace/auto_gpt_workspace",
+  "func_path": "benchmarks.py",
```

Comment:
temporarilly autogpt specific, let's refactor this config.json after this PR is merged, this should be OUT of the agbenchmark folder.